src/components/common/Directors.vue

```diff
@@ -926,6 +926,7 @@ export default class Directors extends Mixins(CommonMixin, DateMixin, DirectorMi
     }
 
     // set form to initial director data again
+    this.director.officer = { ...emptyOfficer }
```

Comment:
This is called after adding a new director or other actions. Either way, the working data (director) needs to be re-initialized after the form was reset otherwise it's undefined.

Reply:
(Apparently Vuetify does not update its v-model if you don't touch the element.)